mews-api.md

```diff
@@ -53,9 +46,9 @@ The property is configured to accept following test credit cards:
 ### Production Environment
 
 * **Platform Address** - `https://www.mews.li`
-* **Client Token** - will be provided to you by Mews following certification in the demo environment \(e.g. `C66EF7B239D24632943D115EDE9CB810-JJ549OU4JF94692C940F6B5A8F9453D`\)
-* **Connection Token** - will be provided to you by Mews or property on request or via [Get Properties](mews-api.md#get-properties) API call \(e.g. `NF9R27B239D24632943D115EDE9CFH3-EA00F8FD8294692C940F6B5A8F9453D`\)
 * **Reservation Push Endpoint** - unique to the environment and listed under [Process Group](mews-api.md#process-group)
+* **Client Token** - will be provided to you by Mews following certification in the test environment \(e.g. `C66EF7B239D24632943D115EDE9CB810-JJ549OU4JF94692C940F6B5A8F9453D`\)
+* **Connection Token** - will be provided to you by Mews or property on request or via [Get Properties](mews-api.md#get-properties) API call \(e.g. `NF9R27B239D24632943D115EDE9CFH3-EA00F8FD8294692C940F6B5A8F9453D`\)
```

Comment:
Should Get Properties be removed?

Reply:
I would keep it for now, but add that this should be done via webhook and link the article from Connector. And also, add that getProperties will be deprecated. What do you think?

Reply:
Since we are going to build a webhook just for channel, I ll keep this part the way it is and update it once we have the webhook ready @allisonchihak @patrikmiksovsky95  ?